src/components/Select/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { PureComponent } from 'react'
+import { string, bool, array, func, object } from 'prop-types'
+import Select, { components } from 'react-select'
+
+export default class SelectInput extends PureComponent {
+  static propTypes = {
+    onChange: func.isRequired,
+    onFocus: func,
+    options: array,
+    placeholder: string,
+    isSearchable: bool,
+    defaultValue: object,
+    components: object,
+  }
+
+  static defaultProps = {
+    options: [],
+    placeholder: 'Select one',
+    isSearchable: false,
+  }
+
+  state = { isOpen: false }
+
+  handleOpenMenu = () =>
+    this.setState({ isOpen: true })
+
+  handleCloseMenu = () =>
+    this.setState({ isOpen: false })
+
+  renderDropdownIndicator = () => {
+    const { isOpen } = this.state
+    const DropdownIndicator = props =>
+      components.DropdownIndicator &&
+        <components.DropdownIndicator {...props}>
+          <Arrow rotate={isOpen} />
+        </components.DropdownIndicator>
+    return DropdownIndicator
+  }
+
+  render () {
+    const {
+      options,
+      placeholder,
+      onChange,
+      onFocus,
+      defaultValue,
+    } = this.props
+
+    return (
+      <Select
+        options={options}
+        styles={this.styles}
+        placeholder={placeholder}
+        onChange={onChange}
+        onFocus={onFocus}
+        className='mc-select'
+        classNamePrefix='mc-select__wrapper'
```

Comment:
I believe this can be `'mc-select'`, as it is the prefix for all classes WITHIN the component.

<img width="284" alt="screen shot 2018-11-26 at 12 11 18 pm" src="https://user-images.githubusercontent.com/249444/49039371-681c8500-f174-11e8-84a5-0eee1cf3c7b0.png">

Reply:
That's because I need one level more of specificity in order to be able to override the default style.
That way I can do 

```
.mc-select .mc-select__wapper {}
```

Using
```
.mc-select .mc-select {}
```
looks weird